### PR TITLE
Add linux metadata

### PR DESCRIPTION
**Pull request by ImUrX**
- Added an svg because linux supports svg icons.
- Added [appstream](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#qsr-app-screenshots-info) metadata for information about the application
- Added a .desktop file for usage in installations

### CODE REVIEW

**Review by TheButlah — APPROVED** [review]

**Comment by TheButlah** [comment]
Note that this PR licenses the metatdata file under CC0

**Review by ttennial94 — APPROVED** [review]

File: dev.slimevr.SlimeVR.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      is a set of open hardware sensors and open source software that facilitates full-body
+      tracking (FBT) in virtual reality. With no base station required, SlimeVR makes wireless
+      VR FBT affordable and comfortable.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">dev.slimevr.SlimeVR.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://media.discordapp.net/attachments/818062236492759050/1044722428951134310/image.png</image>
```
Comment (ButterscotchV) [inline dev.slimevr.SlimeVR.metainfo.xml:48]: I don't like this linking to Discord, can we just put the image on GitHub in this repo and link to it instead?

File: dev.slimevr.SlimeVR.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  <summary>An app for facilitating full-body tracking in virtual reality</summary>
+
+  <!-- CC0 so attribution is not required -->
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT OR Apache-2.0</project_license>
+
+  <content_rating type="oars-1.1" />
+  <url type="homepage">https://slimevr.dev/</url>
+  <url type="bugtracker">https://github.com/SlimeVR/SlimeVR-Server/issues</url>
+  <url type="faq">https://docs.slimevr.dev/server-setup/slimevr-setup.html</url>
```
Comment (ButterscotchV) [inline dev.slimevr.SlimeVR.metainfo.xml:25]: Is this the best link we have for this? I imagine an alternative could be https://docs.slimevr.dev/slimevr101.html.

**Review by ButterscotchV — CHANGES_REQUESTED** [review]

**Review by ButterscotchV — APPROVED** [review]
<3

**Review by Eirenliel — APPROVED** [review]

**Pull request merged**